examples/kafka/docker-compose.yaml

```diff
@@ -0,0 +1,96 @@
+version: '3'
```

Comment:
Suggest to keep kafka setup as simple as possible - e.g. 1 zookeeper. e.g. https://github.com/conduktor/kafka-stack-docker-compose/blob/master/zk-single-kafka-single.yml

Reply:
I wanted to example scenario to be as close to the real word setup as possible.

I prefer to keep it as it is now.